pkg/templates/node_1.10.go

```diff
@@ -0,0 +1,385 @@
+/* vim: set filetype=yaml : */
+
+package templates
+
+var Node_1_10 = `
+passwd:
+  users:
+    - name:          core
+      password_hash: {{ .LoginPassword }}
+{{- if .LoginPublicKey }}
+      ssh_authorized_keys:
+        - {{ .LoginPublicKey | quote }}
+{{- end }}
+locksmith:
+  reboot_strategy: "reboot"
+
+systemd:
+  units:
+    - name: iptables-restore.service
+      enable: true
+    - name: ccloud-metadata-hostname.service
+      enable: true
+      contents: |
+        [Unit]
+        Description=Workaround for coreos-metadata hostname bug
+
+        [Service]
+        ExecStartPre=/usr/bin/curl -s http://169.254.169.254/latest/meta-data/hostname
+        ExecStartPre=/usr/bin/bash -c "/usr/bin/systemctl set-environment COREOS_OPENSTACK_HOSTNAME=$(curl -s http://169.254.169.254/latest/meta-data/hostname)"
+        ExecStart=/usr/bin/hostnamectl set-hostname ${COREOS_OPENSTACK_HOSTNAME}
+        Restart=on-failure
+        RestartSec=30
+        RemainAfterExit=yes
+
+        [Install]
+        WantedBy=multi-user.target
+    - name: docker.service
+      enable: true
+      dropins:
+        - name: 20-docker-opts.conf
+          contents: |
+            [Service]
+            Environment="DOCKER_OPTS=--log-opt max-size=5m --log-opt max-file=5 --ip-masq=false --iptables=false --bridge=none"
+    - name: kubelet.service
+      enable: true
+      contents: |
+        [Unit]
+        Description=Kubelet via Hyperkube ACI
+
+        [Service]
+        Environment="RKT_RUN_ARGS=--uuid-file-save=/var/run/kubelet-pod.uuid \
+          --inherit-env \
+          --dns=host \
+          --net=host \
+          --volume var-lib-cni,kind=host,source=/var/lib/cni \
+          --volume var-log,kind=host,source=/var/log \
```

Comment:
The /etc/machine-id mount is missing we added with #268 to the other templates

Reply:
Added.